pkg/config/runtime.go

```diff
@@ -105,19 +104,15 @@ func (r *Runtime) resolveRules(bag attribute.Bag, aspectSet AspectSet, rules []*
 
 // combined returns a Combined config given an aspect config
 func (r *Runtime) combined(aa *pb.Aspect, aspectSet AspectSet) *Combined {
-	if !aspectSet[aa.GetKind()] {
-		glog.V(3).Infof("Aspect Rejected %v not is set (%v)", aa.GetKind(), aspectSet)
+	if !aspectSet[aa.Kind] {
+		glog.V(3).Infof("Aspect rejected: %v not in set [%v]", aa.Kind, aspectSet)
 		return nil
 	}
 
 	var adp *pb.Adapter
 	// find matching adapter
 	// assume that config references are correct
-	if aa.GetAdapter() != "" {
-		adp = r.adapterByName[aa.GetAdapter()]
-	} else {
-		adp = r.adapterByKind[aa.GetKind()][0]
-	}
+	adp = r.adapterByName[aa.Adapter]
```

Comment:
if would like to specify which metrics I want to collect, I use kind=metrics and configure it.

I do not care where these metrics go, SRE has decided that. Why would you force me to care and specify. 

So we need to assign an adapter for use with an aspect when none is specified. It can de that special adapter named 'default' if you prefer.